def/esprima.js

```diff
@@ -80,13 +80,17 @@ module.exports = function (fork) {
 
     def("ImportDeclaration")
       .bases("Declaration")
-      .build("specifiers", "source")
+      .build("specifiers", "source", "importKind")
       .field("specifiers", [or(
         def("ImportSpecifier"),
         def("ImportNamespaceSpecifier"),
         def("ImportDefaultSpecifier")
       )], defaults.emptyArray)
-      .field("source", def("Literal"));
+      .field("source", def("Literal"))
+      .field("importKind", or(
+        "value",
+        "type"
+      ), defaults["value"]);
```

Comment:
The third argument to `.field` is supposed to be a function that returns a default value when called, so I think this should be `function () { return "value" }` instead of `defaults["value"]`.